chapters/remote/remote-pods.md

```diff
@@ -0,0 +1,42 @@
+# Remote Pods with JinaD
+
+### Prerequisites
+
+Before you start, make sure you have read the [prerequisites for using JinaD](https://docs.jina.ai/chapters/remote/jinad.html#prerequisites).
+
+## Inside a Flow
+
+A common case of using Jina remotely is to have a Flow running locally with Pod on the remote. Following code creates a Flow with a remote Pod on `1.2.3.4` and uploads the pod yml to remote using `upload_files`. All remote logs are collected by the Flow.
+
+#### Python
+
+```python
+from jina import Flow, Document
+f = (Flow()
+     .add(uses='_logforward',
```

Comment:
what is _logforward and why does the user need it

Reply:
This is just an example. `_logforward` helps user print the request.